lib/fast_gettext/translation.rb

```diff
@@ -72,6 +72,12 @@ def ns_(*args, &block)
       # block is called once again to compare result
       block && translation == block.call ? translation : translation.split(NAMESPACE_SEPARATOR).last
     end
+
+    def np_(context, key, *args, &block)
+      options = (args.last.is_a? Hash) ? args.pop : {}
```

Comment:
This works differently the other methods for passing in the separator. Will note the difference in the readme.